src/snowflake/snowpark/modin/plugin/extensions/datetime_index.py

```diff
@@ -335,7 +335,7 @@ def microsecond(self) -> Index:
         --------
         >>> idx = pd.date_range("2000-01-01", periods=3, freq="us")
         >>> idx
-        DatetimeIndex([       '2000-01-01 00:00:00', '2000-01-01 00:00:00.000001',
+        DatetimeIndex(['2000-01-01 00:00:00.000000', '2000-01-01 00:00:00.000001',
```

Comment:
@sfc-gh-azhan I changed the output Index here because `idx.astype(str)` does not give me `['2000-01-01 00:00:00', '2000-01-01 00:00:00.000001', '2000-01-01 00:00:00.000002']` in string format -- astype returns all elements with the nanoseconds part of it.
Is this change alright or should I skip this doctest?

Reply:
Instead of using astype. you can call _format_data on the local native index.